### ISSUE TITLE
Raichu not Duplicating 

### ISSUE THREAD

**Issue by scotthbk** [InertSteak/Pokermon#85]
Hi,

Great work with the mod!

Apologies, if you're already aware of this, I believe that the Raichu joker didn't duplicate for me when I had over $120. 

Best wishes

**Comment by InertSteak** [comment]
Raichu doesn't duplicate itself but turns negative. Did you mean it didn't turn negative?

**Comment by scotthbk** [comment]
Yes, my apologies, it doesn't turn negative.

**Comment by InertSteak** [comment]
Is it your only Raichu? The requirement increases with each one. Also it happens at end of round if you haven't done that yet.

**Comment by scotthbk** [comment]
Yes, it was my only Raichu in a full set with 4 other Joker cards.

**Comment by InertSteak** [comment]
So you had 120$ and nothing happened when on the end of round screen?

**Comment by scotthbk** [comment]
Yes that's right.

**Comment by InertSteak** [comment]
Sorry I can't reproduce this. Could you please record a video or get some screenshots of it not turning negative?

**Comment by InertSteak** [comment]
Closed for inactivity. I don't believe this is an issue any longer.
